resources/views/gallery/index.blade.php

```diff
@@ -2,7 +2,7 @@
 @section('content')
     <h1 class="display-3">Albums aanpassen</h1>
     <div class="justify-content-between d-lg-flex mb-3">
-        <a href="{{ route('galerij.create')}}" class="btn btn-primary">Creeër nieuw album</a>
+        <a href="{{ route('galerij.create')}}" class="btn btn-primary" dusk="create">Creeër nieuw album</a>
```

Comment:
voor de duidelijkheid misschine bij create, edit en delete ff zetten waar het voor is. voorbeeld: createAlbum, editAlbum, deleteAlbum, maakt het net wat overzichtelijker